### ISSUE TITLE
Or tools installed under usr directory 

### ISSUE THREAD

**Issue by tronnlives** [google/or-tools#531]
Hello ,  I have followed the online instructions from the google site to install ortools for python with Ubuntu 17.10 and can only use scripts that import ortools by logging in as usr/root on the command line and actually pasting the code via the command line. When I attempt to open  script regularly it says ortools module not found. I believe it’s because it’s under the root directory and not the home? Does it need to be that way? Can you only use ortools as a root user?

Thanks.

**Comment by lperron** [comment]
How did you install it?
pip install ortools?

Can you remove it, and try it again with

pip install --user ortools



Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00


2017-11-22 15:36 GMT+01:00 tronnlives <notifications@github.com>:

**Comment by tronnlives** [comment]
Thanks for the reply. 

I followed the online directions from the google site.

pip install --upgrade ortools

**Comment by tronnlives** [comment]
I will try what you suggested.

**Comment by tronnlives** [comment]
I got it to work. Thanks! I went back to Windows since I am more comfortable there. Everything seems to be working fine. I have been looking over the quizzes and examples on the google site in efforts to attempt to solve this scheduling problem I have at work. Is the scenario listed below possible to do in ORTOOLs?



Rules to for the schedule

•There are 2 rooms where a minimum of 6 employees are needed in each room.

•Each room has 6 jobs but both have an extra job Incase of a 7th person being scheduled 
 
•Certain jobs can only be done by certain employees 

•Room 2 always gets the extra 7th person unless total staffing is over 15 then room 1 gets the 8th person 

•employees can’t work in the floor consecutively only room#1 can have adjacent days 

•number of jobs has to be spread out as evenly as possible , no worker should the same job 2 days in a row 


•Each room has a 2nd part of the day/shift that also has 6 different jobs.

•certain jobs that are performed in both rooms cannot be paired with the 2nd part of the day jobs 



Monday Minimum staff needed is 12 
Tuesday Minimum staff needed is 11
Wednesday minimum staff needed is 11
Thursday minimum staff needed is 11 
Friday minimum staff needed is 12
Saturday minimum staff needed is 12
Sunday minimum staff needed is 13





Tables Room #1

Machine 
Dumper 
Pkr dumper
Paperwork 
Pkr paperwork 
Chips 


Floor room #2
Supervisor- only a lead can work this if there is 7 people schedules 
Carder-only a lead or dual rate can work this 
Swapper
Cold runner 
Hot runner
Unlocker
Locker
